M2/LICENSING-NOTES.md

```diff
@@ -0,0 +1,122 @@
+| -lmathic           | [mathic](https://github.com/Macaulay2/mathic)                                | LGPL-2.0-or-later                     |
+| -lmathicgb         | [mathicgb](https://github.com/Macaulay2/mathicgb)                            | GPL-2.0-or-later                      |
+| -lmemtailor        | [memtailor](https://github.com/Macaulay2/memtailor)                          | BSD-3-Clause                          |
+| -lmpfi             | [MPFI](https://gitlab.inria.fr/mpfi/mpfi)                                    | LGPL-2.1-or-later                     |
+| -lmpfr             | [MPFR](https://www.mpfr.org/)                                                | LGPL-3.0-or-later                     |
+| -lmps              | [MPSolve](https://numpi.dm.unipi.it/scientific-computing-libraries/mpsolve/) | GPL-3.0-or-later                      |
+| -lnormaliz         | [Normaliz](https://www.normaliz.uni-osnabrueck.de/)                          | GPL-3.0-or-later                      |
+| -lntl              | [NTL](https://libntl.org/)                                                   | LGPL-2.1-or-later                     |
+| -lpython3          | [Python](https://www.python.org/)                                            | PSF-2.0                               |
+| -lreadline         | [Readline](https://www.gnu.org/software/readline)                            | GPL-3.0-or-later                      |
+| -ltbb              | [oneTBB](https://uxlfoundation.github.io/oneTBB/)                            | Apache-2.0                            |
+
+Programs we include with our distribution:
+|                                                                       |                  |
+| --------------------------------------------------------------------- | ---------------- |
+| [4ti2](https://4ti2.github.io/)                                       | GPL-2.0-or-later |
+| [cohomCalg](https://github.com/BenjaminJurke/cohomCalg)               | GPL-3.0-or-later |
+| [CSDP](https://github.com/coin-or/Csdp)                               | EPL-2.0          |
+| [gfan](https://users-math.au.dk/jensen/software/gfan/gfan.html)       | GPL-2.0-or-later |
+| [lrslib](https://cgm.cs.mcgill.ca/~avis/C/lrs.html)                   | GPL-2.0-or-later |
+| [msolve](https://msolve.lip6.fr/)                                     | GPL-2.0-or-later |
+| [nauty](https://pallini.di.uniroma1.it/)                              | Apache-2.0       |
+| [TOPCOM](https://www.wm.uni-bayreuth.de/de/team/rambau_joerg/TOPCOM/) | GPL-2.0-or-later |
+
+
+Libraries linked with some of our programs:
+|        |                                            |                  |
+| ------ | ------------------------------------------ | ---------------- |
+| -lcdd  | [cddlib](https://github.com/cddlib/cddlib) | GPL-2.0-or-later |
+| -lglpk | [GLPK](https://www.gnu.org/software/glpk/) | GPL-3.0-or-later |
+
+Code by other authors compiled as part of M2:
```

Comment:
Why not add Zinin, Furnish, and Boyd to the list below and remove this section?

Reply:
Makes sense!